### PR TITLE
Added documentation for the nft module

### PR DESCRIPTION
**Pull request by croyyds**

### CODE REVIEW

File: docs/chain-details/module_nft.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
+### `nft` module
+
+### Introduction
+
+Fungible tokens are mutually interchangeable with each other, and one most common example of fungible tokens are fiat currencies.
```
Comment (aw126) [inline docs/chain-details/module_nft.md:5]:
```suggestion
Fungible tokens are mutually interchangeable, and one most common example of fungible tokens is fiat currencies.

File: docs/chain-details/module_nft.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+4. a user may `transfer` an NFT as long as he/she is the owner of that NFT.
+
+---
+
+### `issue`:
+Every NFT needs to "live" under a denom: an NFT collection. 
+Therefore, the first step is to issue a denom before one can mint NFTs:
+
+#### `tx nft issue [denom_id] --name [denom_name] --schema [denom_schema] --from [user_address]`- **Issue a denom**
+::: details Example: Issue a new denom with specified name and schema
```
Comment (aw126) [inline docs/chain-details/module_nft.md:76]: May add a paragraph break at the end. "Apart" attach in the same sentence.

**Comment by aw126** [comment]
Looks great.

File: docs/chain-details/module_nft.md (hunk 2 above)
Comment (croyyds) [reply docs/chain-details/module_nft.md:76]: Removed "Apart ..." explanation from the code block section to the NOTE section just after it. Thanks.

**Pull request merged**